web/annotation_editor_params.js

```diff
@@ -75,6 +75,15 @@ class AnnotationEditorParams {
       dispatchEvent("INK_OPACITY", this.valueAsNumber);
     });
     editorStampAddImage.addEventListener("click", () => {
+      this.eventBus.dispatch("reporttelemetry", {
+        source: this,
+        details: {
+          type: "editing",
+          data: {
+            action: "pdfjs.image.add_image_click",
+          },
```

Comment:
Nit: This fits on one line.
```suggestion
          data: { action: "pdfjs.image.add_image_click" },
```